modules/v2/main.tf

```diff
@@ -183,6 +207,11 @@ resource "google_cloud_run_v2_service" "main" {
           }
         }
 
+        env {
```

Comment:
Do we need to provide SA here? App code can use metadata server to retrieve service identity (among other details).

https://cloud.google.com/run/docs/container-contract#metadata-server

Reply:
Removed!